glide.yaml

```diff
@@ -20,6 +20,8 @@ import:
   version: ^2.14
 - package: go.uber.org/yarpc
   version: ^1.21
+- package: github.com/jhump/protoreflect
```

Comment:
You might want to pin this to a commit for now since there are no releases.

Reply:
Pinned. Man looking into this sent me into a spiral of reading about the upcoming v2 api for golang protobuf.